### PR TITLE
Added white_list supported values and usage details

### PR DESCRIPTION
**Pull request by JoseAntonioMHerrera**
## Description

Added supported values to white_list tag in ossec.conf and some other details regarding srcip (the field used to compare against the values in white_list and IPv6 support for IPv4 mapped IPs.

## Checks
- [X] It compiles without warnings.
- [X] Spelling and grammar. 
- [X] Used impersonal speech. 
- [X] Used uppercase only on nouns. 
- [X] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/user-manual/reference/ossec-conf/global.rst [hunk 1]
```diff
@@ -256,17 +256,21 @@ This sets the memory size for the event correlation engine.
 white_list
 ^^^^^^^^^^
 
-This specifies an IP for which Active Responses will not be triggered. Only one IP may be specified for each ``<while_list>`` tag, but several IPs may be used by including multiple ``<white_list>`` tags.
+This specifies an IP, netblock, or hostname for which Active Responses will not be triggered. Only one of those values may
+be specified for each ``<while_list>`` tag, but several values may be used by including multiple ``<white_list>`` tags.
+For this to work the field **srcip** needs to be extracted from the event in the correspondent decoder as it is the one that
+will be use to compare against the values defined in white_list.
```
Comment (Molter73) [inline source/user-manual/reference/ossec-conf/global.rst:262]:
```suggestion
This configuration is compared against the extracted **srcip** field in the alert.
```

File: source/user-manual/reference/ossec-conf/global.rst [hunk 2]
```diff
@@ -256,17 +256,21 @@ This sets the memory size for the event correlation engine.
...
+| **Default value**  | n/a                                                       |
++--------------------+-----------------------------------------------------------+
+| **Allowed values** | Any IP address, netblock (Ej: 192.168.0.0/16) or hostname |
++--------------------+-----------------------------------------------------------+
 
 .. note::
 
-  This option is only valid in server and local installs.
+  This option is only valid in server and local installs. Also, wazuh supports IPv4 mapped into IPv6 if the
+  IPv6 address contains the prefix **::ffff:**, e.g, from ::ffff:192.168.1.17 the ip 192.168.1.17 would be extracted.
```
Comment (Molter73) [inline source/user-manual/reference/ossec-conf/global.rst:273]:
```suggestion
  This option is only valid in server and local installs.
```

File: source/user-manual/reference/ossec-conf/global.rst [hunk 3]
```diff
@@ -256,17 +256,21 @@ This sets the memory size for the event correlation engine.
 white_list
 ^^^^^^^^^^
 
-This specifies an IP for which Active Responses will not be triggered. Only one IP may be specified for each ``<while_list>`` tag, but several IPs may be used by including multiple ``<white_list>`` tags.
+This specifies an IP, netblock, or hostname for which Active Responses will not be triggered. Only one of those values may
```
Comment (Molter73) [inline source/user-manual/reference/ossec-conf/global.rst:259]:
```suggestion
This specifies an IPv4 Address, netblock, IPv4-mapped IPv6 address or hostname for which Active Responses will not be triggered. Only one of those values may
```

File: source/user-manual/reference/ossec-conf/global.rst [hunk 4]
```diff
@@ -256,17 +256,21 @@ This sets the memory size for the event correlation engine.
...
 
-+--------------------+----------------------------+
-| **Default value**  | n/a                        |
-+--------------------+----------------------------+
-| **Allowed values** | Any IP address or netblock |
-+--------------------+----------------------------+
++--------------------+-----------------------------------------------------------+
+| **Default value**  | n/a                                                       |
++--------------------+-----------------------------------------------------------+
+| **Allowed values** | Any IP address, netblock (Ej: 192.168.0.0/16) or hostname |
```
Comment (Molter73) [inline source/user-manual/reference/ossec-conf/global.rst:267]:
```suggestion
| **Allowed values** | Any IPv4 address, IPv4-mapped IPv6 address, netblock (i.e.: 192.168.0.0/16) or hostname |
```

**Review by Molter73 — CHANGES_REQUESTED** [review]

File: source/user-manual/reference/ossec-conf/global.rst (hunk 4 above)
Comment (JoseAntonioMHerrera) [reply source/user-manual/reference/ossec-conf/global.rst]: Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/3925bf788514d8943e6f9aed35b1152cca46b3d5)

File: source/user-manual/reference/ossec-conf/global.rst (hunk 3 above)
Comment (JoseAntonioMHerrera) [reply source/user-manual/reference/ossec-conf/global.rst]: Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/3925bf788514d8943e6f9aed35b1152cca46b3d5)

File: source/user-manual/reference/ossec-conf/global.rst (hunk 2 above)
Comment (JoseAntonioMHerrera) [reply source/user-manual/reference/ossec-conf/global.rst]: Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/3925bf788514d8943e6f9aed35b1152cca46b3d5)

File: source/user-manual/reference/ossec-conf/global.rst (hunk 1 above)
Comment (JoseAntonioMHerrera) [reply source/user-manual/reference/ossec-conf/global.rst]: Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/3925bf788514d8943e6f9aed35b1152cca46b3d5)

File: source/user-manual/reference/ossec-conf/global.rst [hunk 5]
```diff
@@ -256,17 +256,20 @@ This sets the memory size for the event correlation engine.
...
+| **Default value**  | n/a                                                           |
++--------------------+---------------------------------------------------------------+
+| **Allowed values** | Any IPv4 address, netblock (i.e.: 192.168.0.0/16) or hostname |
++--------------------+---------------------------------------------------------------+
 
 .. note::
 
-  This option is only valid in server and local installs.
+  This option is only valid in server and local installs. Also, IPv4-mapped IPv6 addresses are allowed too for
+  comparation.
```
Comment (Molter73) [inline source/user-manual/reference/ossec-conf/global.rst:272]:
```suggestion
  This option is only valid in server and local installs.
  IPv4-mapped IPv6 addresses will be compared against any configured IPv4 or netblock.
```

Comment (JoseAntonioMHerrera) [reply source/user-manual/reference/ossec-conf/global.rst]: Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/395a413280b2ca7ed8a4531e6000a73c1879e6d3)

**Review by Molter73 — APPROVED** [review]
LGTM!

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**